scanners/git-repo-scanner/scanner/git_repo_scanner/github_scanner.py

```diff
@@ -70,8 +70,12 @@ def _process_repos(
         if start_time:
             repos = org.get_repos(type="all", sort="pushed", direction="desc")
 
+        is_over=False
```

Comment:
Please add spaces around the operator (same holds for line 87, 103, 105, 107).

@Weltraumschaf @J12934 you might want to add an optionated code formatter like `ruff` to ensure a consistent formatting.